Project.toml

```
@@ -1,6 +1,6 @@
 name = "JuliaBUGS"
 uuid = "ba9fb4c0-828e-4473-b6a1-cd2560fee5bf"
-version = "0.7.5"
+version = "0.7.6"
```

```suggestion
version = "0.8.0"
```

Should this should be a minor bump? It would break certain invocations of `BUGSModel()`